shared-data/python/opentrons_shared_data/protocol/models/protocol_schema_v6.py

```diff
@@ -156,13 +156,64 @@ class ProtocolSchemaV6(BaseModel):
     robot: Robot
     pipettes: Dict[str, Pipette]
     labware: Dict[str, Labware]
-    labwareDefinitions: Dict[str, LabwareDefinition]
-    commands: List[Command]
     modules: Optional[Dict[str, Module]]
     liquids: Optional[Dict[str, Liquid]]
+    labwareDefinitions: Dict[str, LabwareDefinition]
+    commands: List[Command]
```

Comment:
Might be worth a comment:

```suggestion
    # commands must be after pipettes, labware, etc. for its @validator to work.
    commands: List[Command]
```